src/main/steps/respondent/miam/willingness-to-attend-miam/content.ts

```diff
@@ -21,10 +21,10 @@ const en = {
   miamLabel: miam_how_to_arrange_mediation_label_en,
   threeHint: 'This is a 8 character code',
   summaryText: 'Contacts for help',
-  continue: 'Save and continue',
+  onlyContinue: 'Save and continue',
```

Comment:
Should this value be only 'Continue'?

Reply:
this is label, we can have any value for this